support-frontend/assets/components/checkoutNudge/checkoutNudge.tsx

```diff
@@ -128,13 +131,21 @@ export function CheckoutNudge({
 			<div css={container}>
 				<div css={top}>
 					<div css={topheading}>
-						<h2 css={heading(brand[500], space[2])}>
+						<h2 css={heading(brand[500], space[3])}>
 							{nudgeTitleCopySection1}
 						</h2>
 					</div>
 					<CheckoutNudgeCloseButton onClose={onNudgeClose} />
 				</div>
-				<h2 css={heading(neutral[7], 0)}>{nudgeTitleCopySection2}</h2>
+				<h2
+					css={heading(
+						neutral[7],
+						0,
```

Comment:
This could also be an optional prop for the `heading` css function if you were to use a ternary within the function

Reply:
Struggled with 2 optional parameters, then accessing one or the other. Ended up creating an object with 2 optional parameters inside (line 51). Think it does what you suggest but perhaps there is a better/simpler way?

Reply:
If you make it optional you could do something like this inside the fn: 
`margin-bottom: ${marginBottom ? marginBottom : 0}px;`

Reply:
Also don't think these need to passed in as an object

Reply:
Yep, removed object (was unnecessarily complicated), and applied suggestion